packages/content-conformance/src/index.ts

```diff
@@ -1 +1 @@
-export { ContentConformanceRunner } from './runner.js'
+export * from './runner.js'
```

Comment:
For public exports from a package like this, I think it's preferable if we are explicit:
```suggestion
export { ContentConformanceRunner, RunnerStatus } from './runner.js'
```

Reply:
I was just about to ask for preference! Sounds good